tests/dialects/test_builtin.py

```diff
@@ -167,6 +167,16 @@ def test_IntegerAttr_normalize():
         IntegerAttr(256, 8)
 
 
+def test_IntAttr___bool__():
+    assert not IntAttr(0)
+    assert IntAttr(1)
+
+
+def test_IntegerAttr___bool__():
+    assert not IntegerAttr.from_bool(False)
```

Comment:
```suggestion
    assert not BoolAttr.from_bool(False)
```
Super nit and probably doesn't really matter